features/facebookImport/src/model/importers/utils/importer-util.js

```diff
@@ -21,17 +21,22 @@ async function readJSONFile(relativeFilePath, zipFile) {
     return readFullPathJSONFile(entry);
 }
 
+function decode(s) {
+    let d = new TextDecoder();
+    let a = s.split("").map((r) => r.charCodeAt());
```

Comment:
nit: would prefer full namings for the vars here, to stay consistent with rest file

Reply:
Yep, d can be totally eliminated, too.